src/main/java/magpiebridge/core/analysis/configuration/htmlElement/CheckBox.java

```diff
@@ -0,0 +1,52 @@
+package magpiebridge.core.analysis.configuration.htmlElement;
+
+import magpiebridge.core.analysis.configuration.ConfigurationOption;
+import magpiebridge.core.analysis.configuration.OptionType;
+
+/**
+ * This class represents the Checkbox with additional features.
+ *
+ * @author Ranjith Krishnamurthy
+ */
```

Comment:
Can you please be a bit more specific, what additional features are there? It would be great, if you can also extend the wiki https://github.com/MagpieBridge/MagpieBridge/wiki/Tutorial-5.-MagpieServer-Configuration-Explained